llvm/lib/Target/RISCV/RISCVInstrInfo.h

```diff
@@ -298,6 +298,15 @@ class RISCVInstrInfo : public RISCVGenInstrInfo {
 
   unsigned getTailDuplicateSize(CodeGenOptLevel OptLevel) const override;
 
+  /// Return true if pairing the given load or store may be paired with another.
+  static bool isPairableLdStInstOpc(unsigned Opc);
+
+  static bool isLdStSafeToPair(const MachineInstr &LdSt,
+                               const TargetRegisterInfo *TRI);
+
+  std::optional<std::pair<unsigned, unsigned>>
+  isRVVSpillForZvlsseg(unsigned Opcode) const;
```

Comment:
This looks like a bad merge. `isRVVSpillForZvlsseg` was deleted at some point in the past.

Reply:
Yes, thanks, addressed in https://github.com/llvm/llvm-project/pull/121394